lab5.py

```diff
@@ -0,0 +1,92 @@
+from enum import Enum
+import random
+from datetime import datetime
+
+class Movie:
+    '''
+    this class describes objects of type "Movie" using certain fields
+    '''
+    def __init__(self, id, title, ranking, character_number,
+                 ticket_price, genre, comment ,year, month, day):
+        self.id = id
+        self.title = title
+        self.ranking = ranking
+        self.character_number = character_number
+        self.ticket_price = ticket_price
+        self.comment = comment
+        self.__genre = genre
+        self.__date = datetime(year, month, day)
+        
+    @property
+    def genre(self):
+        return self.__genre
+
+    @genre.setter
+    def genre(self, genre):
+        self.__genre = genre
+    @property
+    def date(self):
+        return self.__date
+
+    @date.setter
+    def date_(self, date):
+        self.__date = date    
+
+class Genre(Enum):
+    '''
+    the Genre class is used to define a enumerate of movie genres using enumerations
+    '''
+    ACTION = 1
+    COMEDY = 2
+    DRAMA = 3
+    FANTASY = 4
+
+class Cinema:
+    '''
+    this class describes objects of type "Cinema" using certain fields
```

Comment:
thi comment should explain the difference between cinema and movie classes for the reader

Reply:
I change this comment